contracts/acl/ACLSyntaxSugar.sol

```diff
@@ -86,6 +86,31 @@ contract ACLSyntaxSugar {
 
 
 contract ACLHelpers {
+
+    enum Op { NONE, EQ, NEQ, GT, LT, GTE, LTE, RET, NOT, AND, OR, XOR, IF_ELSE } // op types
+
+    struct Param {
+        uint8 id;
+        uint8 op;
+        uint240 value; // even though value is an uint240 it can store addresses
+        // in the case of 32 byte hashes losing 2 bytes precision isn't a huge deal
+        // op and id take less than 1 byte each so it can be kept in 1 sstore
+    }
+
+    function encodeParams(Param[] params) internal pure returns (uint256[]) {
+        uint256[] memory encodedParams = new uint256[](params.length);
+
+        for (uint i = 0; i < params.length; i++) {
+            encodedParams[i] = encodeParam(params[i]);
+        }
+
+        return encodedParams;
+    }
+
+    function encodeParam(Param param) internal pure returns (uint256) {
+        return uint256(param.id) << 248 | uint256(param.op) << 240 | param.value;
+    }
```

Comment:
We could also expose the helpers in [`ACLHelpers`](https://github.com/aragon/aragonOS/blob/dev/contracts/test/helpers/ACLHelper.sol#L7).

Reply:
I exposed all the functions except those having `struct` params since they would require the experimental ABI encoder.